src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int side;
+
+    public Square(String color, double side) {
+        super(color);
+        this.side = (int) side;
```

Comment:
Creating a figure, don't pass expressions in the constructor. Create separate variables and pass them on for better code readability. Also, ensure that the value for 'side' is a positive number to avoid creating a square with non-positive sides.